.github/workflows/build_and_test.yml

```diff
@@ -761,82 +1296,35 @@ jobs:
           fi
 
           # first install repo dependencies
-          npm i
-
-          # then micro services dependencies
-          cd ./ee/server/services
-          npm i
-          npm run build
+          yarn
+          yarn build
 
           echo "Building Docker image for service: ${{ matrix.service }}:${IMAGE_TAG}"
 
-          docker build --build-arg SERVICE=${{ matrix.service }} -t rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} .
+          if [[ "${{ matrix.service }}" == "ddp-streamer" ]]; then
+            DOCKERFILE_PATH="./ee/apps/ddp-streamer/Dockerfile"
+          else
+            DOCKERFILE_PATH="./apps/meteor/ee/server/services/Dockerfile"
+          fi
+
+          docker build \
+            --build-arg SERVICE=${{ matrix.service }} \
+            -t rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} \
+            -f ${DOCKERFILE_PATH} \
+            .
 
           docker push rocketchat/${{ matrix.service }}-service:${IMAGE_TAG}
 
           if [[ $GITHUB_REF == refs/tags/* ]]; then
-            if echo "$IMAGE_TAG" | grep -Eq '^[0-9]+\.[0-9]+\.[0-9]+$' ; then
-              RELEASE="latest"
-            elif echo "$IMAGE_TAG" | grep -Eq '^[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+$' ; then
-              RELEASE="release-candidate"
+            RELEASE="${{ needs.release-versions.outputs.release }}"
+
+            if [[ $RELEASE == 'latest' ]]; then
+              if [[ '${{ needs.release-versions.outputs.latest-release }}' == $GITHUB_REF_NAME ]]; then
+                docker tag rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} rocketchat/${{ matrix.service }}-service:${RELEASE}
+                docker push rocketchat/${{ matrix.service }}-service:${RELEASE}
+              fi
+            else
+              docker tag rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} rocketchat/${{ matrix.service }}-service:${RELEASE}
+              docker push rocketchat/${{ matrix.service }}-service:${RELEASE}
             fi
-
-            docker tag rocketchat/${{ matrix.service }}-service:${IMAGE_TAG} rocketchat/${{ matrix.service }}-service:${RELEASE}
-            docker push rocketchat/${{ matrix.service }}-service:${RELEASE}
           fi
-
-  acc-ecr-push:
-    runs-on: ubuntu-latest
-    if: ${{ github.ref == 'refs/heads/develop_pwa' && github.event_name == 'push' }}
```

Comment:
need to reference the develop branch instead of develop_pwa

Reply:
NOTE:  we actually want this disabled for now until we are ready.  Please just add something like branch == "DEACTIVATED"

Reply:
@bhardwajaditya can you address this comment please?

Reply:
@bhardwajaditya I am going to create a PR for this against your branch in your fork if possible.  So don't worry right now about these changes, I'll take care of it and let you know when ready.